pkg/multiboot/multiboot_test.go

```diff
@@ -118,3 +120,38 @@ func TestParseHeader(t *testing.T) {
 		})
 	}
 }
+
+func TestEnsure(t *testing.T) {
+	v = t.Logf
+	v("hi")
```

Comment:
hi

Reply:
well hi to you too

this got fixed
:-)